README.md

```diff
@@ -3,6 +3,12 @@ GOV.UK Frontend ·
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 =====================
 
+> **Warning** We’re currently implementing heavy changes to our code for [our next major version](https://github.com/alphagov/govuk-frontend/milestone/46).
+>
+> We recommend you wait before forking this repository to limit your effort of keeping your fork up to date with our changes.
+>
+> If you were forking to fix a bug in our previous releases, please branch off the [v4.6.0 tag](https://github.com/alphagov/govuk-frontend/tree/v4.6.0).
```

Comment:
Wonder if there's a way to get the `@latest` style tag?

Reply:
Didn't find a URL to point to the latest tag itself, so pointed to the list. All in all, I'm hoping we won't have patch releases to do, but yeah, it's likely we'd forget to update if we did one 🤞🏻